### PR TITLE
3.15.8 amd torque

### PR DESCRIPTION
**Pull request by redhog**
Work around / possible solution for https://github.com/CartoDB/cartodb.js/issues/1298

### CODE REVIEW

**Comment by Cartofante** [comment]
Can one of the admins verify this patch?

File: scripts/mod.torque.footer.js [hunk 1]
```diff
@@ -3,3 +3,6 @@ cartodb.moduleLoad('torque', torque);
 
 Profiler = cartodb.core.Profiler
 
+['require', 'define', 'module'].map(function (name) {
+  window[name] = window.cartotmp[name];
```
Comment (javisantana) [inline scripts/mod.torque.footer.js]: could we delete cartotmp after this so the global namespace is not polluted?

Comment (redhog) [inline scripts/mod.torque.footer.js]: Added a commit that does that :)

**Comment by javisantana** [comment]
thanks @redhog 

@alonsogarciapablo your call

**Comment by rubenmoya** [comment]
As stated in https://github.com/CartoDB/carto.js/issues/1298#issuecomment-404193738, cartodb.js v3 is not under active development. Closing this.

**Pull request closed without merging**